ibft/ibft.go

```diff
@@ -95,13 +88,20 @@ func (i *ibftImpl) StartInstance(opts StartOptions) (bool, int, []byte) {
 	}
 
 	newInstance := NewInstance(InstanceOptions{
-		Logger:         opts.Logger,
-		Me:             i.me,
+		Logger: opts.Logger,
+		Me: &proto.Node{
+			IbftId: opts.Duty.NodeID,
+			Pk:     opts.Duty.PrivateKey.GetPublicKey().Serialize(),
+			Sk:     opts.Duty.PrivateKey.Serialize(),
+		},
 		Network:        i.network,
 		Queue:          i.msgQueue,
 		ValueCheck:     opts.ValueCheck,
 		LeaderSelector: i.leaderSelector,
-		Params:         i.params,
+		Params: &proto.InstanceParams{
```

Comment:
@nivBlox consensus params should be passed in options no?

Reply:
We have passed the params to ibft but now we creating new params from the queue task info

Reply:
default consensus params shouldn't be set here but passed as an argument otherwise you can't change them

Reply:
will change